cmd/zpool/zpool_main.c

```diff
@@ -6703,6 +6708,26 @@ scrub_callback(zpool_handle_t *zhp, void *data)
 	return (err != 0);
 }
 
+int
+rescrub_callback(zpool_handle_t *zhp, void *data)
+{
+	scrub_cbdata_t *cb = data;
+	int err;
+
+	/*
+	 * Ignore faulted pools.
```

Comment:
You can just do `/* Ignore faulted pools. */` here, no need for the multi-line style